operate/client/src/App/ProcessInstance/BottomPanel/VariablePanel/Listeners/styled.ts

```diff
@@ -37,4 +39,13 @@ const WarningFilled = styled(BaseWarningFilled)`
   margin-top: ${spacing01};
 `;
 
-export {Content, StructuredList, CellContainer, WarningFilled};
+const Dropdown = styled(BaseDropdown)`
+  width: 200px;
+  border-bottom: 1px solid var(--cds-border-strong);
```

Comment:
When using stack, there might be no need anymore to set a fixed width. Also the dropdown should already have a bottom border, it looks like it was cut off in the layout and is not visible.

Reply:
Wrapping the elements in a Stack didn't solve either one of these points. If I don't set a width it automatically turns into 100% width. Also I don't think the border is cut off, as when I first developed this I started out having only the dropdown and already had no bottom border.